BlockParty.py

```diff
@@ -198,16 +203,21 @@ def key_action(self, key):
                 if key in self.right_key:
```

Comment:
I would suggest the `else` and `if` to be merged, so it would improve code readability

Reply:
I'm not sure if I correctly interpreted by what you meant by merged, do you want me to change it back to else rather than the elif? But doing that would enable the user to start the game using any key (Supposed to only be with 'Enter')

Reply:
No, I meant L#202 and L#203 on the diff, can be merged to form single `elif` statement; 
i.e [this line](https://github.com/sugarlabs/block-party-activity/pull/12/files#diff-122392eca0444aaa810774cfb9dcd31cR202) and [this](https://github.com/sugarlabs/block-party-activity/pull/12/files#diff-122392eca0444aaa810774cfb9dcd31cR203)

Reply:
Yes, I agree.... Hadn't noticed that! Thanks :)
Will do it tomorrow

Reply:
With the recent changes doing this will logically change how things work. Changing else block to elif blocks will increase the statements checked each time. So I'll let this be :)